core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -149,7 +149,14 @@ private[spark] class Executor(
       tr.kill(interruptThread)
     }
   }
-
+  def killAllTasks (interruptThread: Boolean) : Unit = {
```

Comment:
I could not see this method being called from anywhere. If you don't plan to use it, please remove this.